plugins/services/src/js/events/__tests__/SDKEndpointsActions-test.js

```diff
@@ -85,7 +85,7 @@ describe("SDKEndpointsActions", function() {
   });
 
   describe("#fetchEndpoint", function() {
-    let mockXhr;
+    let mockXhr, originalXhr;
```

Comment:
`originalXhr` is never reassigned and should probably be `const` 👼

Reply:
yes, Thanks.